### ISSUE TITLE
a11y color contrast issue

### ISSUE THREAD

**Issue by belenwebdev** [kirandash/greenhouse-react-ui#9]
There is an accessibility issue due to insufficient color contrast in the primary buttons:

<img width="1440" alt="Screenshot 2023-06-20 at 23 27 48" src="https://github.com/kirandash/greenhouse-react-ui/assets/60265893/0f774f85-b64e-4309-9cae-86d66a3001ff">

**Comment by kirandash** [comment]
@belenwebdev Thank you for raising this. 🙏🏻

**Comment by kirandash** [comment]
@belenwebdev I think the tests are failing. Can you please update the tests to match your changes. Sorry I didn't catch that during my review 

I think I should also create another issue to create a github action that will run test on each PR.

**Comment by kirandash** [comment]
I have created #13 which will make sure to run tests when someone commits. that will help us catch failing tests in future
